docs/api-integration/tlm_event_socket.md

```diff
@@ -0,0 +1,308 @@
+# Tracim Live Message
+
+## Definition of a TLM
+
+The real-time architecture of tracim relies on a socket which dispatch all events generated by users actions.  
+These events are dispatched on a per-user basis and are so called TLM - Tracim Live Messages.
+
+You can connect to socket to listen to tracim activity and trigger actions on any piece of software.
+
+A few use case examples:
+
+- push a file to a web server when a given user comments "VALID FOR PROD" on the content,
+- trigger an email to security staff when someone is invited on a tracim workspace
+- ...
+
+*note: used in combination with REST APIs, the TLM event socket allows to build the real-time client software meeting
+your specific needs.*
+
+### Origin of a TLM
+
+Every change in database that comes from a user creates an `event` line in the database table `events`.  
+See class `Event` in [/backend/tracim_backend/models/event.py](/backend/tracim_backend/models/event.py)).
+
+For each `events`, a list of `messages` are generated.  
+A `message` is the link between a `user` and an `event`.  
+A `message` will be created for each `users` to which Tracim should send the `event`.
+
+A `TLM` is the action of sending the `message` to the frontend, to the logged used.
+
+**Example:**  
+When a content is created in a space, 1 `event` is created.  
+If the space contains 10 users, 10 `messages` are created.  
+If 3 users are currently logged in to Tracim, 3 `TLM` are sent.  
+
+TLM are sent through **Server Sent Event (SSE)** and will trigger the update of the frontend's interface.
+
+## Connection
+
+Using **Server Sent Event (SSE)**, connect to:
+```
+/api/users/<user_id>/live_messages
+```
+Where **<user_id>** is the id of the logged user.
+
+**Example:**
```

Comment:
Do we need to have this example in bold ? (some exemple in this file are not in bold).

Reply:
mmh I think they all are in bold